### DISCUSSION TITLE
Open Source Project

### DISCUSSION THREAD

**Discussion by SusheelThapa** [Schweinepriester/github-profile-achievements#648] [Q&A] [answered]
How to get start contributing in open source project?

**Comment by tauanesales** [comment]
Access any project and make a Pull Request with helpful changes. Try to solve an issue in their repo. Etc

**Comment by tauanesales** [comment, marked as answer]
You can search on GitHub's search bar for [open issues with "good first issue" tag](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+tag%3A%22good+first+issue%22) and start your contributions there on the things you think you might be able to help. 

There are a few guides on how to contribute around if you wanna take a read such as:
- https://github.com/firstcontributions/first-contributions
- https://github.com/wagtail/wagtail/issues/10070


If you want to get started contributing asap, I know a few repositories that would very welcome contributors. Take a look at their issues, their code and what you think you can help out and make a Pull Request:
- https://github.com/ArielMAJ/test.me  (C repository)
- https://github.com/ArielMAJ/Keyboard_Event_Tracker_GUI  (Python Repository)
- https://github.com/Leah9/screengrab  (Python Repository)
- https://github.com/TheFallen-Cat/VocabiPy  (Python Repository)


If this helped you, please add it as an accepted answer.

Ty and GL!
